playwright/pages/dialogs/assignPermissionGroupMembersDialog.ts

```diff
@@ -0,0 +1,29 @@
+import type { Page } from "@playwright/test";
+
+export class AssignPermissionGroupMembersDialog {
+  readonly page: Page;
+
+  constructor(
+    page: Page,
+    readonly searchMembersInput = page.getByTestId("search-members-input").locator("input"),
+    readonly userRow = page.getByTestId("user-row"),
+    readonly assignButton = page.getByTestId("submit"),
+    readonly searchResults = page.getByTestId("search-results"),
+
+  ) {
+    this.page = page;
+  }
+async searchForMembers(memberName: string) {
+  await this.searchMembersInput.first().fill('');
+  await this.searchMembersInput.first().fill(memberName);
```

Comment:
question: what is the purpose of the first fill empty string and then memeberName?